WORKSPACE

```diff
@@ -107,15 +107,28 @@ http_archive(
     ],
 )
 
-# This is a mock version of bazelbuild/rules_python that contains only
-# @rules_python//python:defs.bzl. It is used by protobuf.
-# TODO(#9029): We could potentially replace this with the real @rules_python.
-new_local_repository(
+http_archive(
     name = "rules_python",
-    build_file = "//third_party/rules_python:BUILD",
-    path = "./third_party/rules_python",
-    workspace_file = "//third_party/rules_python:rules_python.WORKSPACE",
+    patch_cmds = EXPORT_WORKSPACE_IN_BUILD_FILE,
+    patch_cmds_win = EXPORT_WORKSPACE_IN_BUILD_FILE_WIN,
+    sha256 = "aa96a691d3a8177f3215b14b0edc9641787abaaa30363a080165d06ab65e1161",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_python/releases/download/0.0.1/rules_python-0.0.1.tar.gz",
+        "https://github.com/bazelbuild/rules_python/releases/download/0.0.1/rules_python-0.0.1.tar.gz",
+    ],
 )
+load("@rules_python//python:repositories.bzl", "py_repositories")
+py_repositories()
+
+# XXXX This is a mock version of bazelbuild/rules_python that contains only
+# @rules_python//python:defs.bzl. It is used by protobuf.
+# TODO(#9029): We could potentially replace this with the real @rules_python.
+#new_local_repository(
+#    name = "rules_python",
+#    build_file = "//third_party/rules_python:BUILD",
+#    path = "./third_party/rules_python",
+#    workspace_file = "//third_party/rules_python:rules_python.WORKSPACE",
+#)
```

Comment:
I think you intended to delete these lines?

Reply:
Yes. I just have not yet because I keep flipping back and forth to figure out the build error.